CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+## Test Your Changes
```

Comment:
Could you add update `README.md` tips? (edit `misc/README.md.erb` and run `rake`)